packages/bsky/service/index.js

```diff
@@ -47,15 +55,22 @@ const main = async () => {
         pathPrefix: cfg.imgUriEndpoint && new URL(cfg.imgUriEndpoint).pathname,
       })
     : undefined
+  const algos = env.feedPublisherDid ? makeAlgos(env.feedPublisherDid) : {}
```

Comment:
i think you're missing an import on `makeAlgos`

Reply:
Ohp, good call!  Exported it to use here but forgot to use it 🙃